biz.aQute.bndlib/src/aQute/bnd/osgi/Builder.java

```diff
@@ -632,6 +632,7 @@ private void doVerify(@SuppressWarnings("unused")
 
 		try (Verifier verifier = new Verifier(this)) {
 			verifier.setFrombuilder(true);
+			verifier.setPedantic(isPedantic());
```

Comment:
I am wondering if this should be in the `Verifier(Analyzer)` constructor or if Verifier should overload `isPedantic` to delegate to the parent Processor?

Reply:
@bjhargrave I added it to `Verifier(Analyzer)`. If you are fine with it, I will finally squash the commits.